packages/collection/src/collection.ts

```diff
@@ -533,6 +591,38 @@ export class Collection<K, V> extends Map<K, V> {
 		return accumulator;
 	}
 
+	/**
+	 * Applies a function to produce a single value. Identical in behavior to
+	 * {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduceRight | Array.reduceRight()}.
+	 *
+	 * @param fn - Function used to reduce, taking four arguments; `accumulator`, `currentValue`, `currentKey`,
```

Comment:
Two names here don't match the names right beneath them.

Reply:
```suggestion
	 * @param fn - Function used to reduce, taking three arguments; `accumulator`, `value`, `collection`,
```